graphics/nxwm/src/ckeyboard.cxx

```diff
@@ -167,7 +167,7 @@ int CKeyboard::open(void)
     {
       // Try to open the keyboard device
 
-      fd = std::open(CONFIG_NXWM_KEYBOARD_DEVPATH, O_RDONLY);
+      fd = open(CONFIG_NXWM_KEYBOARD_DEVPATH, O_RDONLY);
```

Comment:
do we need to change `std::sleep(2);` to `sleep(2);` as well?

Reply:
Yes, done.